client/src/app/editor/file-outputs/file-outputs.component.html

```diff
@@ -31,33 +31,34 @@ <h3 mat-subheader>Output Files</h3>
           <ng-container matColumnDef="actions">
             <mat-header-cell *matHeaderCellDef></mat-header-cell>
             <mat-cell *matCellDef="let row">
-              <button mat-icon-button color="primary" matTooltip="Preview" (click)="openPreview(row)"
-                [disabled]="!isImage(row.name)" title="Preview file"><mat-icon>visibility</mat-icon>
+              <button mat-icon-button color="primary" matTooltip="Preview" (click)="openPreview(row)" [disabled]="!isImage(row.name)" title="Preview file">
+                <mat-icon>visibility</mat-icon>
               </button>
 
-              <button mat-icon-button color="primary" matTooltip="Copy API Link"
-                ngxClipboard [cbContent]="getApiLink(row)"
-                (cbOnSuccess)="copyDone()" (cbOnError)="copyDone(true)"
-                title="Copy API link"><mat-icon>link</mat-icon>
+              <button mat-icon-button color="primary" matTooltip="Copy API Link" ngxClipboard [cbContent]="getApiLink(row)" (cbOnSuccess)="copyDone()"
+                (cbOnError)="copyDone(true)" title="Copy API link">
+                <mat-icon>link</mat-icon>
               </button>
 
-              <a mat-icon-button color="primary" matTooltip="Download" [attr.download]="row.name"
-                target="_blank" [href]="row.download_url"
-                title="Download file"><mat-icon>file_download</mat-icon>
+              <a mat-icon-button color="primary" matTooltip="Download" [attr.download]="row.name" target="_blank" [href]="row.download_url"
+                title="Download file">
+                <mat-icon>file_download</mat-icon>
               </a>
             </mat-cell>
           </ng-container>
 
           <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
-          <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
+          <mat-row *matRowDef="let row; columns: displayedColumns;"   [@staggerIn]="'showing'"></mat-row>
         </mat-table>
       </ng-container>
     </ng-container>
     <ng-template #noOutputMessage>
-      <div class="ml-no-information-message">
-        <mat-icon class="ml-no-information-message-icon">info_outline</mat-icon>
-        <h4 class="ml-no-information-message-headline">No outputs available</h4>
-        <p class="ml-no-information-message-text">Select an execution that has generated output files.</p>
```

Comment:
We actually want to keep this message. Everytime a user selects an execution without outputs, we show this message.